### PR TITLE
Topic/run vmtests improvements

### PR DESCRIPTION
**Pull request by danielocfb**
Please see individual commits for descriptions. Only grouped together because of GitHub limitations.

### CODE REVIEW

**Comment by chantra** [comment]
LGTM

File: travis-ci/vmtest/run_selftests.sh [hunk 1]
```diff
@@ -26,25 +26,31 @@ test_progs() {
 		echo "test_progs:$?" >> "${STATUS_FILE}"
 		foldable end test_progs
 	fi
```
Comment (mykola-lysenko) [inline travis-ci/vmtest/run_selftests.sh:28]: I assume it is correct, but will double check. Does it mean that we execute only no_alu32 tests each time? All other tests are being executed depending on the kernel version?

Comment (mykola-lysenko) [reply travis-ci/vmtest/run_selftests.sh:28]: E.g. on latest we execute all the tests, on 5.* we execute prog_tests and no_alu_32, and on 4.9.0 we execute only no_alu32

Comment (danielocfb) [reply travis-ci/vmtest/run_selftests.sh:28]: Yep, that seems to be the case.

Comment (danielocfb) [reply travis-ci/vmtest/run_selftests.sh:28]: That behavior is unchanged from earlier. E.g., check the selftest runs on https://github.com/libbpf/libbpf/pull/546.

Comment (mykola-lysenko) [reply travis-ci/vmtest/run_selftests.sh:28]: oh, I know, it is just something that caught my eye

**Review by mykola-lysenko — APPROVED** [review]

**Pull request merged**